### ISSUE TITLE
issue with bigslice

### ISSUE THREAD

**Issue by sherinesaber** [zellerlab/GECCO#5]
Hello I am trying to run the second line and it is not working :
 gecco convert gbk -i bigslice_dir/dataset_1/U00096.2_cluster_1/ --format bigslice

```
x An unexpected error occurred. Consider opening a new issue on the bug tracker ( https://github.com/zellerlab/GECCO/issues/new ) if it persists, including the traceback below:
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│                                                                                                  │
│ /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gecco/cli/commands │
│                                                                                                  │
│   150 │   │   │   │   subcmd.quiet = self.quiet                                                  │
│   151 │   │   │   │   subcmd.progress.disable = self.quiet > 0                                   │
│   152 │   │   │   # run the subcommand                                                           │
│ ❱ 153 │   │   │   return subcmd.execute(ctx)                                                     │
│   154 │   │   except CommandExit as sysexit:                                                     │
│   155 │   │   │   return sysexit.code                                                            │
│   156 │   │   except KeyboardInterrupt:                                                          │
│ /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gecco/cli/commands │
│                                                                                                  │
│   195 │   │   │   # run the appropriate method                                                   │
│   196 │   │   │   if self.args["gbk"]:                                                           │
│   197 │   │   │   │   if self.args["--format"] == "bigslice":                                    │
│ ❱ 198 │   │   │   │   │   self._convert_gbk_bigslice(ctx)                                        │
│   199 │   │   │   │   elif self.args["--format"] == "fna":                                       │
│   200 │   │   │   │   │   self._convert_gbk_fna(ctx)                                             │
│   201 │   │   │   │   elif self.args["--format"] == "faa":                                       │
│                                                                                                  │
│ /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gecco/cli/commands │
│                                                                                                  │
│    89 │   │   # load the original coordinates from the `*.clusters.tsv` files                    │
│    90 │   │   coordinates = {}                                                                   │
│    91 │   │   types = {}                                                                         │
│ ❱  92 │   │   for cluster_file in self.progress.track(cluster_files, task_id=task, precision="") │
│    93 │   │   │   cluster_fh = ctx.enter_context(open(cluster_file))                             │
│    94 │   │   │   for row in ClusterTable.load(cluster_fh):                                      │
│    95 │   │   │   │   ty = ";".join(sorted(ty.name for ty in row.type.unpack()))                 │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: track() got an unexpected keyword argument 'precision'
```

**Comment by althonos** [comment]
Hi @sherinesaber , thanks for the report! It looks like I forgot to update this part of the code when I did my last refactor, I know where the issue is coming from and I'll make a new patch release ASAP.

**Comment by althonos** [comment]
Just released `v0.8.4`, please update GECCO and it should work now!

**Comment by sherinesaber** [comment]
can I ask how to update  ?
